Add `--pages-branch` parameter

michalbiesek opened the pull request:
Ref:
  helm/chart-releaser@378f527
  helm/chart-releaser@cb0563b

cpanato reviewed (changes requested):
need rebase

and maybe we only set that parameter if the user pass it otherwise we keep the existing behavior, wdyt?

michalbiesek commented:
> need rebase

Rebased.

> and maybe we only set that parameter if the user pass it otherwise we keep the existing behavior, wdyt?

I've changed the implementation. Please take a look.

cpanato commented on README.md:
```diff
@@ -22,6 +22,7 @@ A GitHub action to turn a GitHub project into a self-hosted Helm chart repo, usi
 - `skip_existing`: Skip package upload if release/tag already exists
 - `mark_as_latest`: When you set this to `false`, it will mark the created GitHub release not as 'latest'.
 - `packages_with_index`: When you set this to `true`, it will upload chart packages directly into publishing branch.
+- `pages_branch`: The GitHub pages branch
```
```suggestion
- `pages_branch`: Name of the branch to be used to push the index and artifacts. (default to: gh-pages but it is not set in the action it is a default value for the chart-releaser binary)
```

michalbiesek commented on action.yml:
```diff
@@ -35,6 +35,9 @@ inputs:
     description: "Upload chart packages directly into publishing branch"
     required: false
+  pages_branch:
+    description: "The GitHub pages branch"
```
Done.

michalbiesek commented:
> need rebase

Rebased.

cpanato commented:
thanks!
